tests/nimbleclientbase.py

```diff
@@ -49,6 +51,29 @@
 os_client = None
 
 
+def is_array_version_above_or_equal(arr_version_to_check):
```

Comment:
have added a logic wherein we now check the array version against which the testcases are running.. there are few testcase which are not supported on lower version of nimOS. hence , this is needed